mgr/ConsDefs.pm

```diff
@@ -983,7 +983,8 @@
 
     $XMLINCDIR = `$xml --cflags`;
     chomp($XMLINCDIR);
-    $XMLINCDIR =~ s/-I//;
+    $XMLINCDIR =~ s/-I//g;
+    $XMLINCDIR =~ s/ /:/g;
```

Comment:
Without knowing what `--cflags` can return, I wonder if a more robust approach is to
```suggestion
    $XMLINCDIR =~ s/ -I/:/g;
    $XMLINCDIR =~ s/-I//g;
```
Or to even pattern-match to only select `-I` flags?

Reply:
Makes sense. I like it, thank you!